Makefile

```diff
@@ -117,6 +118,9 @@ else ifeq ($(GRUCODE),f3dzex) # Fast3DZEX (2.08J / Animal Forest - Dōbutsu no M
 else ifeq ($(GRUCODE),super3d) # Super3D
   $(warning Super3D is experimental. Try at your own risk.)
   DEFINES += SUPER3D_GBI=1 F3D_NEW=1
+else ifeq ($(GRUCODE),f3dex3) # Fast3DEX3
+  DEFINES += F3DEX_GBI_3=1 F3DEX_GBI_SHARED=1
+  $(warning Fast3DEX3 is experimental. Try at your own risk.)
```

Comment:
I'd get rid of this warning personally, _especially_ if f3dex3 is default. That should exist as a comment if at all.

Reply:
See above reply.

Reply:
Actually super risky apparently; will reluctantly accept keeping this in (as much as I like stderr being completely devoid of output)

Reply:
Marking this as resolved since currently I can only get this working as a "warning".